libafl_targets/src/libfuzzer.c

```diff
@@ -47,7 +47,7 @@ EXT_FUNC_IMPL(main, int, (int argc, char **argv), false) {
   #endif
 }
 
-  #if defined(_WIN32)
+  #if defined(_WIN32) && !defined(__clang__)
```

Comment:
not sure if this doesn't break libfuzzer_windows_asan.
i added the tests in [#2961](https://github.com/AFLplusplus/LibAFL/pull/2949) let's see